common/src/main/java/com/wynntils/screens/maps/AbstractMapScreen.java

```diff
@@ -76,8 +83,11 @@ public abstract class AbstractMapScreen extends WynntilsScreen {
     // Zooming updates zoomLevel, but we also cache zoomRenderScale for rendering
     protected float zoomLevel = MapRenderer.DEFAULT_ZOOM_LEVEL;
     protected float zoomRenderScale = MapRenderer.getZoomRenderScaleFromLevel(zoomLevel);
+    protected BoundingBox mapBoundingBox = BoundingBox.EMPTY;
 
+    // TODO: This is not used anymore. It's only here to make the code compile.
```

Comment:
... and you can't remove the code that still references it?

Reply:
All of the "mouseClicked" methods use it. You know, that big mess of "keybinds" :). I'll be working on unifying that, after the marker options are added, and we figured out feature/category overrides.